packages/SwingSet/src/controller.js

```diff
@@ -95,8 +98,7 @@ function makeEvaluate(e) {
   });
 }
 
-function buildSESKernel(hostStorage) {
-  // console.log('transforms', transforms);
+function makeSESEvaluator() {
```

Comment:
I'm still studying this, but I'm worried that `makeSESEvaluator()` is going to cause the creation of multiple SES root realms, in a way that will cause identity-discontinuity problems. Things will change when we get to new SES, but in current SES I think we want to have exactly one SES root realm (unless `--no-ses`, in which case we'll have zero SES root realms).

Reply:
makeSESEvaluator is called where buildSESKernel was called previously in buildVatController. buildVatController should only be called once in buildVatController (in controller.js), which is called by start.js or its equivalent. I think this is safe and there's no change.

Reply:
Ah, now I see it.. much nicer. `buildSESKernel` only builds the kernel, just like `buildNonSESKernel`. Awesome.